modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -84,7 +84,17 @@ export default {
     }
   },
   mounted() {
-    this.menuItems = [ ...this.items ];
+    this.menuItems = [
+      ...this.items.map(item => {
```

Comment:
Not totally sure the spread is necessary. A map returns a new array so I think you could do something like:
```
this.menuItems = this.items.map(item => { ... });
```
and get the same result.

Reply:
true.